src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/ApplicationsOrchestrator.cs

```diff
@@ -63,7 +69,7 @@ public async Task<ApplicationViewModel> GetApplication(ApplicationRequest reques
             {
                 return null;
             }
-
+            var isOwnerOrTransactor = _userService.IsOwnerOrTransactor(request.AccountId);
```

Comment:
Had some trouble testing this so correct me if I'm wrong but it looks like the controller still has the manage account policy on it so viewers wouldn't be able to view this page.